### PR TITLE
feat: updated README.md; added terraforms files

### PR DESCRIPTION
**Pull request by owlleg6**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,51 @@
-# Google <> Terraform module
-Terraform module for creation Google <>
+# Google firewall Terraform module
+Terraform module for creation Google firewall
```
Comment (owlleg6) [inline README.md]: "of"

File: README.md [hunk 2]
```diff
@@ -1,12 +1,51 @@
...
+| <a name="input_env"></a> [env](#input\_env)                                               | Variable to mark the environment of the resource (used to create services) | `string` | "dev"   |   yes    |
+| <a name="input_network"></a> [network](#input\_network)                                   | The name or self_link of the network to attach this firewall to            | `string` | n/a     |   yes    |
+| <a name="input_sql_instance_ip"></a> [sql\_instance\_ip](#input\_sql\_instance\_ip)       | The SQL database ip address to connect to                                  | `string` | n/a     |   yes    |
+| <a name="input_sql_port"></a> [sql\_port](#input\_sql\_port)                              | The database port number to connect to                                     | `string` | "1433"  |    no    |
+| <a name="input_dataproc_range"></a> [dataproc\_range](#input\_dataproc\_range)            | CIDR range for Dataproc subnet                                             | `string` | n/a     |   yes    |
+
+## Outputs
+
+NO outputs.
```
Comment (owlleg6) [inline README.md]: "No outputs"

File: variables.tf [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+variable "product_base_name" {
+  description = "Cloud resources base name (used to create services)"
+  type        = string
+}
+
+variable "env" {
+  description = "Variable to mark the environment of the resource (used to create services)."
+  type        = string
+  default     = "dev"
```
Comment (owlleg6) [inline variables.tf]: env is required, remove default value please

File: README.md [hunk 4]
```diff
@@ -1,12 +1,51 @@
...
+Terraform module for creation Google firewall
 
 ## Usage
 
 <!-- BEGIN_TF_DOCS -->
+## Requirements
 
+| Name                                                                      | Version   |
+| ------------------------------------------------------------------------- | --------- |
+| <a name="requirement_terraform"></a> [terraform](#requirement\_terraform) | >= 1.0.0  |
```
Comment (owlleg6) [inline README.md]: random provider?

File: README.md [hunk 5]
```diff
@@ -1,12 +1,51 @@
...
+| <a name="requirement_google"></a> [google](#requirement\_google)          | >= 4.24.0 |
+
+## Providers
+
+| Name                                                             | Version   |
+| ---------------------------------------------------------------- | --------- |
```
Comment (owlleg6) [inline README.md]: random?

Comment (ivasm-ko) [reply README.md]: done

File: README.md (hunk 4 above)
Comment (ivasm-ko) [reply README.md]: done

File: variables.tf (hunk 3 above)
Comment (ivasm-ko) [reply variables.tf]: done

File: README.md (hunk 2 above)
Comment (ivasm-ko) [reply README.md]: done

File: README.md (hunk 1 above)
Comment (ivasm-ko) [reply README.md]: done

**Pull request merged**